dotCMS/src/main/webapp/html/portlet/ext/common/edit_permissions_tab_inc.jsp

```diff
@@ -104,7 +104,7 @@
 </div>
 
 <!-- START Saving permissions dialog -->
-	<div id="savingPermissionsDialog" dojoType="dijit.Dialog" title="<%= LanguageUtil.get(pageContext, "saving-permissions") %>" style="display: none;">
+	<div id="savingPermissionsDialog" data-dojo-props="closable:false" dojoType="dijit.Dialog" title="<%= LanguageUtil.get(pageContext, "saving-permissions") %>" style="display: none;">

```

Comment:
Why do we set `data-dojo-props="closable:false"` and in the other `divs`?

Reply:
previously we are hiding dialog close button using javascript but in  dojo dialog have prop `data-dojo-props="closable:false"` for hiding the close button.